ERROR - 2024-07-12 14:58:39 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'tienda', 3306, '', 0)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tienda\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tienda\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tienda\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\tienda\app\Validation\Userrules.php(24): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\tienda\system\Validation\Validation.php(311): App\Validation\Userrules->existsUser('1205150087', 'usuario', Array, NULL, 'usuario')
#8 C:\xampp\htdocs\tienda\system\Validation\Validation.php(197): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', '1205150087', Array, Array)
#9 C:\xampp\htdocs\tienda\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#10 C:\xampp\htdocs\tienda\app\Controllers\Usuario.php(35): CodeIgniter\Controller->validate(Array, Array)
#11 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Usuario->login()
#12 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#13 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tienda\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tienda\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tienda\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\tienda\app\Validation\Userrules.php(24): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\tienda\system\Validation\Validation.php(311): App\Validation\Userrules->existsUser('1205150087', 'usuario', Array, NULL, 'usuario')
#7 C:\xampp\htdocs\tienda\system\Validation\Validation.php(197): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', '1205150087', Array, Array)
#8 C:\xampp\htdocs\tienda\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\tienda\app\Controllers\Usuario.php(35): CodeIgniter\Controller->validate(Array, Array)
#10 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Usuario->login()
#11 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#12 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-07-12 14:58:39 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `usuario` = :usuario:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Validation\Userrules.php(24): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\Validation\Validation.php(311): App\Validation\Userrules->existsUser('1205150087', 'usuario', [...], null, 'usuario')
 7 SYSTEMPATH\Validation\Validation.php(197): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', '1205150087', [...], [...])
 8 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Usuario.php(35): CodeIgniter\Controller->validate([...], [...])
10 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuario->login()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-07-12 15:31:56 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'tienda', 3306, '', 0)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tienda\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tienda\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tienda\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\tienda\app\Validation\Userrules.php(24): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\tienda\system\Validation\Validation.php(311): App\Validation\Userrules->existsUser('1205150087', 'usuario', Array, NULL, 'usuario')
#8 C:\xampp\htdocs\tienda\system\Validation\Validation.php(197): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', '1205150087', Array, Array)
#9 C:\xampp\htdocs\tienda\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#10 C:\xampp\htdocs\tienda\app\Controllers\Usuario.php(35): CodeIgniter\Controller->validate(Array, Array)
#11 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Usuario->login()
#12 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#13 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tienda\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tienda\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tienda\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\tienda\app\Validation\Userrules.php(24): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\tienda\system\Validation\Validation.php(311): App\Validation\Userrules->existsUser('1205150087', 'usuario', Array, NULL, 'usuario')
#7 C:\xampp\htdocs\tienda\system\Validation\Validation.php(197): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', '1205150087', Array, Array)
#8 C:\xampp\htdocs\tienda\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\tienda\app\Controllers\Usuario.php(35): CodeIgniter\Controller->validate(Array, Array)
#10 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Usuario->login()
#11 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#12 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-07-12 15:31:56 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `usuario` = :usuario:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Validation\Userrules.php(24): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\Validation\Validation.php(311): App\Validation\Userrules->existsUser('1205150087', 'usuario', [...], null, 'usuario')
 7 SYSTEMPATH\Validation\Validation.php(197): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', '1205150087', [...], [...])
 8 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Usuario.php(35): CodeIgniter\Controller->validate([...], [...])
10 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuario->login()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-07-12 15:43:08 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\producto\listitemcarts.php on line 3.
 1 APPPATH\Views\producto\listitemcarts.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php', 3)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('producto/listitemcarts', [], true)
 5 APPPATH\Controllers\Carrito.php(28): view('producto/listitemcarts', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Carrito->viewProductosGrid()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Carrito))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-07-12 15:44:42 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\producto\listitemcarts.php on line 3.
 1 APPPATH\Views\producto\listitemcarts.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php', 3)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('producto/listitemcarts', [], true)
 5 APPPATH\Controllers\Carrito.php(28): view('producto/listitemcarts', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Carrito->viewProductosGrid()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Carrito))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-07-12 19:47:23 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\producto\listitemcarts.php on line 3.
 1 APPPATH\Views\producto\listitemcarts.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php', 3)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('producto/listitemcarts', [], true)
 5 APPPATH\Controllers\Carrito.php(28): view('producto/listitemcarts', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Carrito->viewProductosGrid()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Carrito))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-07-12 23:24:27 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\producto\listitemcarts.php on line 3.
 1 APPPATH\Views\producto\listitemcarts.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php', 3)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('producto/listitemcarts', [], true)
 5 APPPATH\Controllers\Carrito.php(28): view('producto/listitemcarts', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Carrito->viewProductosGrid()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Carrito))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-07-12 23:25:07 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\producto\listitemcarts.php on line 3.
 1 APPPATH\Views\producto\listitemcarts.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php', 3)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('producto/listitemcarts', [], true)
 5 APPPATH\Controllers\Carrito.php(28): view('producto/listitemcarts', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Carrito->viewProductosGrid()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Carrito))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-07-12 23:29:25 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\producto\listitemcarts.php on line 3.
 1 APPPATH\Views\producto\listitemcarts.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php', 3)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('producto/listitemcarts', [], true)
 5 APPPATH\Controllers\Carrito.php(28): view('producto/listitemcarts', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Carrito->viewProductosGrid()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Carrito))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-07-12 23:29:46 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\producto\listitemcarts.php on line 3.
 1 APPPATH\Views\producto\listitemcarts.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php', 3)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('producto/listitemcarts', [], true)
 5 APPPATH\Controllers\Carrito.php(28): view('producto/listitemcarts', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Carrito->viewProductosGrid()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Carrito))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-07-12 23:29:48 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'tienda2', 3306, '', 0)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tienda\app\Models\categoria_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat('...')
#4 C:\xampp\htdocs\tienda\app\Controllers\Categoria.php(16): App\Models\categoria_model->getListado()
#5 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Categoria->getjson_ListadoCategorias('')
#6 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categoria))
#7 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tienda\app\Models\categoria_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat('...')
#3 C:\xampp\htdocs\tienda\app\Controllers\Categoria.php(16): App\Models\categoria_model->getListado()
#4 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Categoria->getjson_ListadoCategorias('')
#5 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categoria))
#6 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-07-12 23:29:48 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'tienda2'
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\categoria_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat(\'r_\',id_categoria) as dt_rowid, id_categoria as id, nombre_categoria as descripcion  FROM categorias  order by id_categoria;')
 3 APPPATH\Controllers\Categoria.php(16): App\Models\categoria_model->getListado()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Categoria->getjson_ListadoCategorias('')
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Categoria))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-07-12 23:29:48 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'tienda2', 3306, '', 0)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tienda\app\Models\marca_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat('...')
#4 C:\xampp\htdocs\tienda\app\Controllers\Marca.php(16): App\Models\marca_model->getListado()
#5 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Marca->getjson_ListadoMarcasCB('')
#6 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Marca))
#7 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tienda\app\Models\marca_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat('...')
#3 C:\xampp\htdocs\tienda\app\Controllers\Marca.php(16): App\Models\marca_model->getListado()
#4 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Marca->getjson_ListadoMarcasCB('')
#5 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Marca))
#6 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-07-12 23:29:48 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'tienda2'
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\marca_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat(\'r_\',id_marca) as dt_rowid, id_marca as id, nombre_marca as descripcion  FROM marcas  order by id_marca;')
 3 APPPATH\Controllers\Marca.php(16): App\Models\marca_model->getListado()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Marca->getjson_ListadoMarcasCB('')
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Marca))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-07-12 23:29:49 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'tienda2', 3306, '', 0)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tienda\app\Models\producto_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat('...')
#4 C:\xampp\htdocs\tienda\app\Controllers\Producto.php(40): App\Models\producto_model->getListado()
#5 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Producto->getjson_ListadoProductos('data')
#6 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Producto))
#7 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tienda\app\Models\producto_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat('...')
#3 C:\xampp\htdocs\tienda\app\Controllers\Producto.php(40): App\Models\producto_model->getListado()
#4 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Producto->getjson_ListadoProductos('data')
#5 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Producto))
#6 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-07-12 23:29:49 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'tienda2'
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\producto_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat(\'r_\',id) as dt_rowid, c.nombre_categoria as idcategoria, m.nombre_marca as idmarca, p.id, p.descripcion, precio, pvp, impuesto, foto 
         FROM producto p, categorias c, marcas m where p.idcategoria = c.id_categoria and p.idmarca = m.id_marca  order by id;')
 3 APPPATH\Controllers\Producto.php(40): App\Models\producto_model->getListado()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Producto->getjson_ListadoProductos('data')
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Producto))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-07-12 23:29:52 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'tienda2', 3306, '', 0)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tienda\app\Models\producto_model.php(16): CodeIgniter\Database\BaseConnection->query('SELECT c.id_cat...')
#4 C:\xampp\htdocs\tienda\app\Models\producto_model.php(49): App\Models\producto_model->execQuery('SELECT c.id_cat...')
#5 C:\xampp\htdocs\tienda\app\Controllers\Producto.php(29): App\Models\producto_model->getListadoGrid('0', '0')
#6 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Producto->viewProductosGrid('0', '0', '1')
#7 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Producto))
#8 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#10 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tienda\app\Models\producto_model.php(16): CodeIgniter\Database\BaseConnection->query('SELECT c.id_cat...')
#3 C:\xampp\htdocs\tienda\app\Models\producto_model.php(49): App\Models\producto_model->execQuery('SELECT c.id_cat...')
#4 C:\xampp\htdocs\tienda\app\Controllers\Producto.php(29): App\Models\producto_model->getListadoGrid('0', '0')
#5 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Producto->viewProductosGrid('0', '0', '1')
#6 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Producto))
#7 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2024-07-12 23:29:52 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'tienda2'
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\producto_model.php(16): CodeIgniter\Database\BaseConnection->query('SELECT c.id_categoria as categoria_id, c.nombre_categoria as categoria, m.nombre_marca as marca, p.*
        FROM producto p, marcas m, categorias  c
        WHERE m.id_marca = p.idmarca and c.id_categoria = p.idcategoria  order by c.nombre_categoria, m.nombre_marca, p.descripcion')
 3 APPPATH\Models\producto_model.php(49): App\Models\producto_model->execQuery('SELECT c.id_categoria as categoria_id, c.nombre_categoria as categoria, m.nombre_marca as marca, p.*
        FROM producto p, marcas m, categorias  c
        WHERE m.id_marca = p.idmarca and c.id_categoria = p.idcategoria  order by c.nombre_categoria, m.nombre_marca, p.descripcion')
 4 APPPATH\Controllers\Producto.php(29): App\Models\producto_model->getListadoGrid('0', '0')
 5 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Producto->viewProductosGrid('0', '0', '1')
 6 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Producto))
 7 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-07-12 23:29:53 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'tienda2', 3306, '', 0)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tienda\app\Models\producto_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat('...')
#4 C:\xampp\htdocs\tienda\app\Controllers\Producto.php(40): App\Models\producto_model->getListado()
#5 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Producto->getjson_ListadoProductos('data')
#6 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Producto))
#7 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tienda\app\Models\producto_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat('...')
#3 C:\xampp\htdocs\tienda\app\Controllers\Producto.php(40): App\Models\producto_model->getListado()
#4 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Producto->getjson_ListadoProductos('data')
#5 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Producto))
#6 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2024-07-12 23:29:53 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'tienda2'
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Models\producto_model.php(23): CodeIgniter\Database\BaseConnection->query('SELECT Concat(\'r_\',id) as dt_rowid, c.nombre_categoria as idcategoria, m.nombre_marca as idmarca, p.id, p.descripcion, precio, pvp, impuesto, foto 
         FROM producto p, categorias c, marcas m where p.idcategoria = c.id_categoria and p.idmarca = m.id_marca  order by id;')
 3 APPPATH\Controllers\Producto.php(40): App\Models\producto_model->getListado()
 4 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Producto->getjson_ListadoProductos('data')
 5 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Producto))
 6 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2024-07-12 23:29:58 --> Error connecting to the database: mysqli_sql_exception: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', 'root', '', 'tienda2', 3306, '', 0)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tienda\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\tienda\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\tienda\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#6 C:\xampp\htdocs\tienda\app\Validation\Userrules.php(24): CodeIgniter\BaseModel->first()
#7 C:\xampp\htdocs\tienda\system\Validation\Validation.php(311): App\Validation\Userrules->existsUser('cristian', 'usuario', Array, NULL, 'usuario')
#8 C:\xampp\htdocs\tienda\system\Validation\Validation.php(197): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'cristian', Array, Array)
#9 C:\xampp\htdocs\tienda\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#10 C:\xampp\htdocs\tienda\app\Controllers\Usuario.php(35): CodeIgniter\Controller->validate(Array, Array)
#11 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Usuario->login()
#12 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#13 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#14 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Unknown database 'tienda2' in C:\xampp\htdocs\tienda\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tienda\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tienda\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\tienda\system\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\tienda\system\BaseModel.php(662): CodeIgniter\Model->doFirst()
#5 C:\xampp\htdocs\tienda\app\Validation\Userrules.php(24): CodeIgniter\BaseModel->first()
#6 C:\xampp\htdocs\tienda\system\Validation\Validation.php(311): App\Validation\Userrules->existsUser('cristian', 'usuario', Array, NULL, 'usuario')
#7 C:\xampp\htdocs\tienda\system\Validation\Validation.php(197): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'cristian', Array, Array)
#8 C:\xampp\htdocs\tienda\system\Controller.php(152): CodeIgniter\Validation\Validation->run()
#9 C:\xampp\htdocs\tienda\app\Controllers\Usuario.php(35): CodeIgniter\Controller->validate(Array, Array)
#10 C:\xampp\htdocs\tienda\system\CodeIgniter.php(942): App\Controllers\Usuario->login()
#11 C:\xampp\htdocs\tienda\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
#12 C:\xampp\htdocs\tienda\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#13 C:\xampp\htdocs\tienda\index.php(79): CodeIgniter\CodeIgniter->run()
#14 {main}
CRITICAL - 2024-07-12 23:29:58 --> Unable to connect to the database.
Main connection [MySQLi]: Unknown database 'tienda2'
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `usuario` = :usuario:
 LIMIT 1', [...], false)
 3 SYSTEMPATH\Model.php(264): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(662): CodeIgniter\Model->doFirst()
 5 APPPATH\Validation\Userrules.php(24): CodeIgniter\BaseModel->first()
 6 SYSTEMPATH\Validation\Validation.php(311): App\Validation\Userrules->existsUser('cristian', 'usuario', [...], null, 'usuario')
 7 SYSTEMPATH\Validation\Validation.php(197): CodeIgniter\Validation\Validation->processRules('usuario', 'usuario', 'cristian', [...], [...])
 8 SYSTEMPATH\Controller.php(152): CodeIgniter\Validation\Validation->run()
 9 APPPATH\Controllers\Usuario.php(35): CodeIgniter\Controller->validate([...], [...])
10 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Usuario->login()
11 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuario))
12 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2024-07-12 23:30:17 --> foreach() argument must be of type array|object, null given
in APPPATH\Views\producto\listitemcarts.php on line 3.
 1 APPPATH\Views\producto\listitemcarts.php(3): CodeIgniter\Debug\Exceptions->errorHandler(2, 'foreach() argument must be of type array|object, null given', 'C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php', 3)
 2 SYSTEMPATH\View\View.php(228): include('C:\\xampp\\htdocs\\tienda\\app\\Views\\producto\\listitemcarts.php')
 3 SYSTEMPATH\View\View.php(231): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1190): CodeIgniter\View\View->render('producto/listitemcarts', [], true)
 5 APPPATH\Controllers\Carrito.php(28): view('producto/listitemcarts', [...])
 6 SYSTEMPATH\CodeIgniter.php(942): App\Controllers\Carrito->viewProductosGrid()
 7 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Carrito))
 8 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
